src/routes/v2/authenticated/review.ts

```diff
@@ -185,6 +191,21 @@ export class ReviewsRouter {
       description
     )
 
+    // Step 5: Create notifications
+    collaborators.forEach(async (user: User & { SiteMember: SiteMember }) => {
+      // Don't send notification to self
+      if (user.id.toString() === userWithSiteSessionData.isomerUserId) return
+      const notificationType = reviewersMap[user.email || ""]
```

Comment:
should we even create if the user is a github user?

Reply:
It shouldn't, but this route is only accessible to email users right?